test/fixtures/get-test-database.ts

```diff
@@ -94,6 +94,7 @@ export const getTestDatabase = async (
     connect_webview_ids: [cw.connect_webview_id],
     connected_account_ids: [ca.connected_account_id],
     user_identifier_key: "seed_client_session_user",
+    user_identity_ids: ["john_user_identity_id"],
```

Comment:
Where does this id come from? I would not expect it to be hard coded here.

Reply:
This value is available when we seed the database in https://github.com/seamapi/fake-seam-connect/blob/d163139d5c3a13d24dd96d9fc5fc006c0e316f60/src/lib/database/seed.ts#L43 so I was just adding to this client session as well